instance/models/openedx_appserver.py

```diff
@@ -176,6 +176,10 @@ class Meta:
             "behind a firewall. (In the django admin, separate group names with a comma.)"
         )
     )
+    enable_vm_dns_records = models.BooleanField(
+        default=False,
+        help_text="Set additional DNS records pointing directly to the active app servers."
+    )
```

Comment:
@antoviaque I completely understand, and as I said above, I expect that we will enable this by default in the future.  However, given the tight timebox of the ticket this was developed on, I preferred to play it safe for now.